frontend/src/components/common/List/SelectInput.tsx

```diff
@@ -3,14 +3,23 @@ import { getIn, useFormikContext } from 'formik';
 import React from 'react';
 
 interface ISelectInputProps<SelectTypes> {
+  /** Specifies that the HTML element should be disabled */
   disabled?: boolean;
+  /** The option to show on the drop-down by default */
   defaultKey?: keyof SelectTypes;
+  /** The field name */
   field: string;
+  /** The underlying HTML element to use when rendering the FormControl */
+  as?: React.ElementType;
+  /** optional help text to display below the FormControl */
+  helpText?: string;
```

Comment:
I see a helpText prop here, but it doesn't seem to be used.

Reply:
It is a passthrough all the way down to the **Input** (passing thru **InputGroup**)